config/filter.d/horde.conf

```diff
@@ -4,7 +4,7 @@
 [Definition]
 
 
-failregex = ^ HORDE \[error\] \[(horde|imp)\] FAILED LOGIN for \S+ \[<HOST>\](\(forwarded for \[\S+\]\))? to (Horde|{[^}]+}) \[(pid \d+ )?on line \d+ of \S+\]$
+failregex = ^ ERR: HORDE \[(horde|imp)\] FAILED LOGIN for \S+ to (horde|{[^}]+}) \((<HOST>)\) \[(pid \d+ )?on line \d+ of \S+\]$
```

Comment:
Please add second failregex after your new one (for the backwards compatibility to previous versions)...
```INI
failregex = ^ ERR: HORDE ... (1st regex)
            ^ HORDE ... (2nd regex)
```